### PR TITLE
Fix-73341 Installing extension from extension page "Install" button keeps the notification baloon in bottom right

### LINKED ISSUE #73341: Installing extension from extension page "Install" button keeps the notification baloon in bottom right
Issue Type: <b>Bug</b>

I noticed when I install extension by clicking on the green "Install" button from within the VScode extension page, as opposed to the "Would you like to install..." notification/chat bubble in the lower right corner of the editor, the chat bubble stays there, asking me if I want to install the extension I just installed.

Would be great if the state was in sync and the bubble would be dismissed when I installed the extension, no matter how.

<details>
<summary>Screenshot after installation</summary>
Screen Shot 2019-05-06 at 17 54 50

</details>

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:22:55.268Z)
OS version: Darwin x64 18.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.79GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|10.5.1
solargraph|cas|0.19.6
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|1.8.2
es7-react-js-snippets|dsz|2.2.2
gitlens|eam|9.7.1
prettier-vscode|esb|1.9.0
rails-snippets|Hri|1.0.8
rest-client|hum|0.21.2
vscode-codemetrics|kis|1.18.0
vscode-language-babel|mgm|0.0.22
ruby-rubocop|mis|0.8.0
sublime-keybindings|ms-|4.0.0
debugger-for-chrome|msj|4.11.3
JavaScriptSnippets|nat|0.2.0
ruby|reb|0.22.3
code-settings-sync|Sha|3.2.9
change-case|wma|1.0.0
JavaScriptSnippets|xab|1.7.2

</details>

### PR DESCRIPTION
**Pull request by skprabhanjan**
@sandy081 , Removed the notification bubble as discussed to fix #73341.
Please review this and let me know :)

### CODE REVIEW

File: src/vs/workbench/contrib/extensions/node/extensionsWorkbenchService.ts [hunk 1]
```diff
@@ -1001,29 +1001,9 @@ export class ExtensionsWorkbenchService extends Disposable implements IExtension
 			if (extension) {
 				return this.windowService.focusWindow()
 					.then(() => this.open(extension));
+			} else {
+				return Promise.resolve(null);
 			}
-
-			return this.queryGallery({ names: [extensionId], source: 'uri' }, CancellationToken.None).then(result => {
```
Comment (sandy081) [inline src/vs/workbench/contrib/extensions/node/extensionsWorkbenchService.ts]: I am afraid you removed too much here. This breaks the functionality if the extension is not installed. If the extension is not installed, you should query from gallery and open the extension. You should only remove the prompt (1015, 1023) after opening the extension.

**Review by sandy081 — CHANGES_REQUESTED** [review]

**Comment by skprabhanjan** [comment]
@sandy081 , Sorry for that , Did not check that part.
Changed it accordingly :)

**Comment by sandy081** [comment]
@skprabhanjan looks like your change has build failures. Can you please fix them.
I guess it is caused because of unused import `Severity`.

**Comment by dmolin** [comment]
> @skprabhanjan looks like your change has build failures. Can you please fix them.
> I guess it is caused because of unused import `Severity`.

@sandy081 , It doesn't look like it. Looking at Azure pipelines it seems to be a platform/architecture issue on the machine where the compilation of the code was taking place:

```
d:\a\1\s\remote\node_modules\keytar\node_modules\nan\nan.h(24): fatal error C1083: Cannot open include file: 'node_version.h': No such file or directory (compiling source file ..\src\async.cc) [D:\a\1\s\remote\node_modules\keytar\build\keytar.vcxproj]
```

**Comment by sandy081** [comment]
Could be a random error in windows.

**Comment by skprabhanjan** [comment]
@sandy081  , so what should I do on this from my end now ?

**Comment by skprabhanjan** [comment]
@sandy081 , Any update on this PR ? 
The  build should be good now.

**Comment by sandy081** [comment]
Will take a look in next milestone.

**Comment by skprabhanjan** [comment]
@sandy081 , this should be good to be merged now correct ? ( if the PR has passed your review)

**Comment by skprabhanjan** [comment]
@sandy081 , any updates on this ? Sorry

**Review by sandy081 — APPROVED** [review]

**Pull request merged**